stylelint-polaris/configs/coverage.js

```diff
@@ -14,14 +14,18 @@ module.exports = {
     'declaration-property-value-disallowed-list': [
       {
         display: ['grid', 'flex'],
+        // No numeric values
         top: [/\$.+/],
         bottom: [/\$.+/],
         left: [/\$.+/],
         right: [/\$.+/],
         width: [/\$.+/],
         height: [/\$.+/],
+        // zero and one only, no numeric values or additions
```

Comment:
This currently allows `0`, `1`, and custom properties: https://regex101.com/r/s8IpNe/1